helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -1025,6 +1026,34 @@ public void dropResource(String clusterName, String resourceName) {
     accessor.removeProperty(keyBuilder.resourceConfig(resourceName));
   }
 
+  @Override
+  public void addCloudConfig(String clusterName, CloudConfig cloudConfig) {
```

Comment:
This interface design is a little bit awkward. If we already have a cluster name in the CloudConfig, why cannot we just rely on that one? Now we might have 2 different clusterNames, and the caller is required to read the document for understanding where to put the name...

Reply:
Fixed. Cluster name has been removed.